cmake/SetupPackages.cmake

```diff
@@ -52,6 +52,11 @@ if (RAJA_ENABLE_OPENMP)
   endif()
 endif()
 
+if (RAJA_ENABLE_CLANG_CUDA)
+  set(RAJA_ENABLE_CUDA On)
```

Comment:
The `RAJA_ENABLE_CLANG_CUDA` is a gross solution but it allows us to be able to build both NVCC with a clang host compiler, and CUDA with clang natively.